msal-client-credentials-flow/app.js

```diff
@@ -0,0 +1,59 @@
+/*
+ This application demonstrates how to issue a call to a protected API
+ using the client credentials flow.  A request will be issued to
+ Microsoft Graph using the application's own identity.
+*/
+
+// Microsoft Authentication Library (MSAL) for Node.js
+const msal = require('@azure/msal-node')
+
+// Node.js Express Framework
+const express = require('express')
+
+// Used to make the HTTP GET request to the Graph API
+const https = require('https')
+
+// MSAL configuration
+const msalConfig = {
+auth: {
+  // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+  clientId: '',
+  // Client secret 'Value' (not the ID) from 'Client secrets' in app registration in Azure portal
+  clientSecret: '',
+  // Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+  authority: ''
+}
+}
```

Comment:
Can you run the formatter on this file, looks like this isn't indented properly for example.

Reply:
Resolved.